pulpcore/cli/common/generic.py

```diff
@@ -1191,21 +1192,23 @@ def callback(
     ) -> None:
         ctx.obj = repository_ctx.get_version_context()
 
-    callback.add_command(list_command(decorators=decorators))
-    callback.add_command(show_command(decorators=decorators + [version_option]))
-    callback.add_command(destroy_command(decorators=decorators + [version_option]))
-
-    @callback.command()
-    @repository_option
-    @version_option
-    @pass_repository_version_context
-    @pass_pulp_context
-    def repair(
-        pulp_ctx: PulpCLIContext,
-        repository_version_ctx: PulpRepositoryVersionContext,
-    ) -> None:
-        result = repository_version_ctx.repair()
-        pulp_ctx.output_result(result)
+    callback.add_command(list_command(decorators=decorators + [content_in_option]))
```

Comment:
This is also adding the `--content` fields to all of the other plugin repo version commands too, neat.

Reply:
yes!